applications/audio/broadcast_configuration_tool/src/main.c

```diff
@@ -378,105 +438,432 @@ static int cmd_list(const struct shell *shell, size_t argc, char **argv)
 	return 0;
 }
 
-static int cmd_start(const struct shell *shell, size_t argc, char **argv)
+static int adv_create_and_start(const struct shell *shell, uint8_t big_index)
 {
 	int ret;
 
 	size_t ext_adv_buf_cnt = 0;
 	size_t per_adv_buf_cnt = 0;
 
-	if (argc < 2) {
-		shell_error(shell, "Usage: bct start <preset>");
-		return -EINVAL;
+	/* Get advertising set for BIG0 */
+	ret = ext_adv_populate(big_index, &ext_adv_data[big_index], ext_adv_buf[big_index],
+			       ARRAY_SIZE(ext_adv_buf[big_index]), &ext_adv_buf_cnt);
+	if (ret) {
+		return ret;
 	}
 
-	struct bt_bap_lc3_preset *preset = preset_find(argv[1]);
+	ret = per_adv_populate(big_index, &per_adv_data[big_index], &per_adv_buf[big_index], 1,
+			       &per_adv_buf_cnt);
+	if (ret) {
+		return ret;
+	}
 
-	if (!preset) {
-		shell_error(shell, "Preset not found");
-		return -EINVAL;
+	/* Start broadcaster */
+	ret = bt_mgmt_adv_start(big_index, ext_adv_buf[big_index], ext_adv_buf_cnt,
+				&per_adv_buf[big_index], per_adv_buf_cnt, false);
+	if (ret) {
+		shell_error(shell, "Failed to start advertising for BIG%d", big_index);
+		return ret;
 	}
 
-	struct broadcast_source_big broadcast_param;
+	return 0;
+}
 
-	broadcast_create(&broadcast_param, preset);
+static int cmd_start(const struct shell *shell, size_t argc, char **argv)
+{
+	int ret;
 
-	ret = broadcast_source_enable(&broadcast_param, 1);
-	if (ret) {
-		shell_error(shell, "Failed to enable broadcaster(s): %d", ret);
-		return ret;
+	if (argc == 2) {
+		size_t big_index = strtoul(argv[1], NULL, 10);
+
+		if (big_index >= CONFIG_BT_ISO_MAX_BIG) {
+			shell_error(shell, "BIG index out of range");
+			return -EINVAL;
+		}
+
+		ret = broadcast_source_enable(&broadcast_param[big_index], big_index);
+		if (ret) {
+			shell_error(shell, "Failed to enable broadcaster: %d", ret);
+			return ret;
+		}
+
+		ret = adv_create_and_start(shell, big_index);
+
+	} else {
+		for (int i = 0; i < CONFIG_BT_ISO_MAX_BIG; i++) {
+			if (broadcast_param[i].subgroups != NULL ||
+			    broadcast_param[i].num_subgroups > 0) {
+				ret = broadcast_source_enable(&broadcast_param[i], i);
+				if (ret) {
+					shell_error(shell, "Failed to enable broadcaster(s): %d",
+						    ret);
+					return ret;
+				}
+				ret = adv_create_and_start(shell, i);
+				if (ret) {
+					return ret;
+				}
+			}
+		}
 	}
 
-	/* Get advertising set for BIG0 */
-	ret = ext_adv_populate(0, &ext_adv_data[0], ext_adv_buf[0], ARRAY_SIZE(ext_adv_buf[0]),
-			       &ext_adv_buf_cnt);
+	return 0;
+}
+
+static int broadcaster_stop(const struct shell *shell, uint8_t big_index)
```

Comment:
Note to self: Is there a broadcaster_stop_all?

Reply:
Yes, it works the same was as bct start, either specify what BIG to start/stop or do it for all. Will add some more usage prints explaining this